lib/engine/game/g_1835/meta.rb

```diff
@@ -16,6 +16,13 @@ module Meta
         GAME_RULES_URL = 'http://google.com'
 
         PLAYER_RANGE = [3, 7].freeze
+        OPTIONAL_RULES = [
+          {
+            sym: :Clemens,
```

Comment:
style - we tend to keep symbols snake case

Reply:
ok, i'll correct it tomorrow.